shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-spi/src/main/java/org/apache/shardingsphere/data/pipeline/api/config/rulealtered/HandleConfiguration.java

```diff
@@ -51,6 +51,9 @@
     
     private List<String> jobShardingDataNodes;
     
+    /**
+     * Logic tables. Separated by ','. e.g. t_order,t_order_item.
+     */
```

Comment:
There is unnecessary to add JavaDoc for private filed

Reply:
JavaDoc removed.